cypress/integration/old-bugs/no-empty-date-in-generator.spec.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * In #507, we noticed that an empty date in the generator causes an error in 'My requests' as parsing the empty date
+ * will fail.
+ */
+
+describe('Requests without a date should not be allowed', () => {
+    it('Deleting the date should reset to today', () => {
+        cy.visit('/generator');
+
+        cy.contains('Information block').click();
+        cy.get('#request-date').clear();
+
+        cy.contains('Send email').click();
+        cy.get('.dropdown-container').contains('Copy text manually').should('be.visible');
+        cy.contains('Copy text manually').click({ force: true });
+
+        cy.get('#reference')
+            .invoke('val')
+            .then((reference) =>
+                cy
+                    .window()
+                    .then((win) =>
+                        win.accessLocalForageStore('my-requests').then((instance) => {
+                            instance
+                                .getItem(`${reference}-access`)
+                                .then((i) => expect(i.date).to.equal(new Date().toISOString().substring(0, 10)));
```

Comment:
Let's see if we ever manage to run the test on midnight ;)

Reply:
Good catch. But eh, I'll only care enough to fix that if that ever actually happens… :D